### ISSUE TITLE
Deprecate `Passthru` parameter on `Get-WebFile` PowerShell helper

### ISSUE THREAD

**Issue by AdmiringWorm** [chocolatey/choco#2640]
## Is Your Feature Request Related To A Problem? Please describe.

In version 0.9.10 the parameter `Passthru` was changed to a recommendation of not using it, and had been left due to backwards compatibility.
As this is recommended to not be used, we should deprecate its use and output a warning if it is used with a pending removal in the next major version.

## Describe The Solution. Why is it needed?

It makes sense to deprecate anything that we do recommend to not be used, instead of purely leaving it as a comment.

## Additional Context.

Link to DO NOT USE comment: https://github.com/chocolatey/choco/blob/19d7dd10d2cb4392afd157d52ccc7067f69611e7/src/chocolatey.resources/helpers/functions/Get-WebFile.ps1#L60-L61

## Related Issues

TODO: Add a link to an issue that will eventually remove this deprecation.

**Comment by gep13** [comment]
We decided that this wasn't going out with 1.0.0, right? Should we assign to 1.1.0 so that it doesn't get lost?

**Comment by AdmiringWorm** [comment]
Yeah, we decided not to deprecate it in v1, I've changed the milestone to 1.1.0.
